libraries/Adafruit_LittleFS/src/Adafruit_LittleFS.cpp

```diff
@@ -55,6 +56,14 @@ Adafruit_LittleFS::~Adafruit_LittleFS ()
 // Return true if mounted successfully else probably corrupted.
 // User should format the disk and try again
 bool Adafruit_LittleFS::begin (struct lfs_config * cfg)
+{
+  while (pdTRUE != xSemaphoreTake(_mutex,  portMAX_DELAY)) {}
```

Comment:
I think we don't need to check the return with `portMAX_DELAY` since it literally means wait forever.

Reply:
I did not realize that there were no other error conditions here.  You are correct again.